src/index.ts

```diff
@@ -113,7 +120,7 @@ function createTerminal() {
     assert: (assertion: boolean, obj: any) => assertion && send('assert', obj),
     error: (obj: any) => send('error', obj),
     info: (obj: any) => send('info', obj),
-    log: (obj: any) => send('log', obj),
+    log: (...obj: any[]) => send('log', obj),
```

Comment:
We can use `send('log',...obj)` and inside send check if there are several objects, then we stringify and join them and send a single one. You should join with `.join(' ')` as this is what `console.log(obj1, obj2)` does in the browser. Also, let's use a simple `JSON.string(obj)` without indentation if there are several objects so we keep the message as a single line

Reply:
Where do you say to use JSON.string(obj)? I used stringify when there is only 1 object with identation of 2 because it was already being used by the log before of this. When we have more than one object we use toString to keep all in one line

Reply:
Check the commits I made 👍🏼